### PR TITLE
Branch entry add

### LINKED ISSUE #70: Refactor code so that schedule and tasks are under one class, Task

### PR DESCRIPTION
**Pull request by lrj689**
Add Entry Command. START_DATE is optional, if not provided will be set to be the same as end date

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/AddEntryCommand.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            + "[" + PREFIX_START_DATE + "START DATE] "
+            + PREFIX_END_DATE + "END DATE "
+            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_NAME + "consultation 1 "
+            + PREFIX_START_DATE + "2021-04-01 17:00 "
+            + PREFIX_END_DATE + "2021-04-01 19:00 "
+            + PREFIX_TAG + "CS2100";
+
+    public static final String MESSAGE_SUCCESS = "New entry added: %1$s";
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/logic/commands/AddEntryCommand.java:33]: should we move these messages to commons.core.Messages? similar approach can be used for #94 #95

File: src/main/java/seedu/address/logic/parser/AddEntryCommandParser.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+        EntryName entryName = ParserUtil.parseEntryName(argMultimap.getValue(PREFIX_NAME).get());
+        EntryDate endDate = ParserUtil.parseEntryDate(argMultimap.getValue(PREFIX_END_DATE).get());
+        try {
+            startDate = ParserUtil.parseEntryDate(argMultimap.getValue(PREFIX_START_DATE).get());
+        } catch (NoSuchElementException e) {
+            startDate = endDate;
+        }
+        Set<Tag> tagList = ParserUtil.parseTags(argMultimap.getAllValues(PREFIX_TAG));
+
```
Comment (nicholastanvis) [inline src/main/java/seedu/address/logic/parser/AddEntryCommandParser.java:54]: should we add an assert statement here that throws `if endDate.isBefore(startDate)`

**Review by nicholastanvis — CHANGES_REQUESTED** [review]

**Review by tsh22 — APPROVED** [review]
lgtm

**Review by nicholastanvis — APPROVED** [review]
thanks for addressing the comments, approved

**Review by goatygoatygoat — APPROVED** [review]

**Pull request merged**